packages/common/src/components/DeactivatedAccount.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import { useTranslation } from "next-i18next";
+import styled from "styled-components";
+import IconButton from "./IconButton";
+import { IconArrowRight } from "hds-react";
+import { fontBold, fontRegular } from "../common/typography";
+
+const ErrorContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: var(--container-width-xl);
+  margin: 0 auto;
+  gap: 42px;
+  padding-block: 42px;
+  padding-inline: var(--spacing-s);
+  --spacing-hz: var(--spacing-s);
+  img {
+    width: 185px;
+    height: auto;
+    margin: 0 auto;
+  }
+  @media (min-width: 768px) {
+    flex-direction: row;
+    padding-inline: var(--spacing-m);
+    --spacing-hz: var(--spacing-m);
+    img {
+      display: flex;
+      margin: 0;
+      margin-inline: auto;
+      width: auto;
+      max-height: 418px;
+      max-width: 350px;
```

Comment:
not a fan of these magic numbers. where are they from? (except from Figma for this particular page)

Reply:
Yeah, they're "magic" numbers - and are based on the image dimensions in Figma.